include/natalie.hpp

```diff
@@ -106,7 +106,7 @@ ArrayObject *block_args_to_array(Env *env, size_t signature_size, size_t argc, V
 
 void arg_spread(Env *env, size_t argc, Value *args, const char *arrangement, ...);
 
-std::pair<Value, Value> coerce(Env *, Value, Value);
+std::pair<Value, Value> coerce(Env *, Value, Value, bool raise = true);
```

Comment:
Can we make this an enum like we did with [ConstLookupFailureMode](https://github.com/seven1m/natalie/blob/master/include/natalie/object.hpp#L43-L46)?

Reply:
Yeah of course! That's a great idea 😄

Reply:
I added it! The name I chose is a bit mouthful to me but I couldn't think of a better one...